test/memory/psahabu/lineno-memLeak.good

```diff
@@ -0,0 +1,7 @@
+
+=================================================================================================================
+Allocated Memory (Bytes)         Number   Size     Total    Description                      Address             
+=================================================================================================================
+lineno-memLeak.chpl:2            1        16       16       MyClass                          0x00007fa2edc03f70  
```

Comment:
You'll have to put a prediff of some sort, since the address will change run-to-run. Also, we almost certainly only care about the .chpl:2 part. Probably grepping for it in the prediff is the way to go.